examples/private_set_intersection/client/private_set_intersection.cc

```diff
@@ -68,7 +68,12 @@ int main(int argc, char** argv) {
 
   // TODO(#1066): Use a more restrictive Label, based on a bearer token shared between the two
   // clients.
-  oak::label::Label label = oak::PublicUntrustedLabel();
+  std::string public_key_base64 = "f41SClNtR4i46v2Tuh1fQLbt/ZqRr1lENajCW92jyP4=";
+  std::string public_key;
+  if (!absl::Base64Unescape(public_key_base64, &public_key)) {
+    LOG(FATAL) << "Could not decode public key: " << public_key_base64;
+  }
+  oak::label::Label label = oak::WebAssemblyModuleSignatureLabel(public_key);
```

Comment:
Could we have part of the example that uses an invalid key too? Please make sure it runs as part of CI too, either as example or as test.